packages/external-db-bigquery/lib/bigquery_schema_provider.js

```diff
@@ -1,7 +1,10 @@
 const { CollectionDoesNotExists } = require('velo-external-db-commons').errors
 const { translateErrorCodes } = require('./sql_exception_translator')
 const SchemaColumnTranslator = require('./sql_schema_translator')
-const { SystemFields, validateSystemFields, parseTableData } = require('velo-external-db-commons')
+const { SystemFields, validateSystemFields, parseTableData, SchemaOperations } = require('velo-external-db-commons')
+
+const { LIST, LIST_HEADERS, CREATE, DROP, ADD_COLUMN, REMOVE_COLUMN, DESCRIBE_COLLECTION } = SchemaOperations
+const schemaSupportedOperations =  [LIST, LIST_HEADERS, CREATE, DROP, ADD_COLUMN, REMOVE_COLUMN, DESCRIBE_COLLECTION]
```

Comment:
@MXPOL  please extract those function to schema_utils or something like that, i don't want the schema provider to expose them, they are util functions that shouldn't be here